easybuild/easyconfigs/g/GLib/GLib-2.42.1-GCC-4.9.2.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = 'GLib'
+version = '2.42.1'
+
+homepage = 'http://www.gtk.org/'
+description = """GLib is one of the base libraries of the GTK+ project"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://ftp.gnome.org/pub/gnome/sources/glib/%(version_major_minor)s/']
+sources = ['glib-%(version)s.tar.xz']
+
+dependencies = [
+    ('libffi', '3.2.1'),
+    ('gettext', '0.19.4'),
+]
+builddependencies = [('Python', '2.7.9', '', ('foss', '2015a'))]
```

Comment:
hmm, hold on... just noticed this

@gjbex: shouldn't we be using a `bare` Python which was built with only `GCC/4.9.2` here instead?

Having a dependency using a full toolchain in something that uses a compiler-only toolchain is a little bit weird...